pkg/mcp/server.go

```
@@ -137,14 +121,46 @@ func (m *MCPHealthServer) Start() error {
 }
 
 // RegisterTool registers a new tool on the MCPHealthServer
 func (m *MCPHealthServer) RegisterTool(t *mcp.Tool, handler mcp.ToolHandlerFor[any, any]) {
 	mcp.AddTool(m.server, t, handler)
 }
 
+// healthzHandler responds with 200 OK for liveness and readiness probes.
+func healthzHandler(w http.ResponseWriter, _ *http.Request) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	fmt.Fprintln(w, "ok")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)